app/dashboard/volunteers/page.tsx

```diff
@@ -1,3 +1,28 @@
-export default function Test() {
-  return <>Hello</>;
+import { createClient } from "@/utils/supabase/server";
+import { Volunteer, columns } from "./columns";
+import { DataTable } from "./data-table";
+
+async function getData(): Promise<Volunteer[]> {
+  const supabase = createClient();
+  let { data: volunteer, error } = await supabase.from("volunteer").select("*");
+
+  if (error) {
+    console.error(error);
+    return [];
+  }
+
+  return volunteer || [];
```

Comment:
Use the null resolution operator `??` in cases like these. Moreover, this check is unnecessary as the user of the hook shouldn't need to differentiate between empty `data` and non-empty `data`.